spec/time_of_day_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe Timely::TimeOfDay do
+  subject(:time_of_day) { Timely::TimeOfDay.new(4, 52) }
+
+  its(:hour) { should == 4 }
+  its(:minute) { should == 52 }
+
+  context 'asking for that time on a date' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.